benchmarks/benchmarks/stats.py

```diff
@@ -57,44 +57,71 @@ def time_ttest_ind_diff_var(self):
 
 
 class Distribution(Benchmark):
+    # add distributions here
+    dists = ['cauchy', 'gamma', 'beta']
+
     param_names = ['distribution', 'properties']
     params = [
-        ['cauchy', 'gamma', 'beta'],
-        ['pdf', 'cdf', 'rvs', 'fit']
+        dists, ['pdf', 'logpdf', 'cdf', 'logcdf', 'rvs', 'fit', 'sf', 'logsf',
+                'ppf', 'isf', 'moment', 'stats_s', 'stats_v', 'stats_m', 'stats_k', 'stats_mvsk', 'entropy', 'median', 'mean',
+                'var', 'std', 'interval', 'expect']
     ]
+    distcont = dict(distcont)
+    # maintain previous benchmarks' values
+    custom_input = {'gamma': [5], 'beta': [5, 3]}
 
     def setup(self, distribution, properties):
-        np.random.seed(12345678)
-        self.x = np.random.rand(100)
+        self.dist = getattr(stats, distribution)
+      
+        shapes = self.distcont[distribution]
+
+        if isinstance(self.dist, stats.rv_discrete):
+            self.isCont = False
+            kwds = {'loc': 4}
+        else: 
+            self.isCont = True
+            kwds = {'loc': 4, 'scale': 10}
```

Comment:
This is to allow discrete distributions to run, even though we don't have any of them in this benchmark yet.

Reply:
Please go ahead and add them all (discrete and continuous) so we start getting data and can track it over time.

Reply:
Please call my attention back to this when that's ready, and I'll probably merge.